### PR TITLE
PXP-500: [CLI] Implement the logs command

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

screenshot (30)

Ticket: [PXP-500](https://mindvalley.atlassian.net/browse/PXP-500)

### Added
- [x] generate google logging v2 api from protobuf using `tonic_build`
- [x] add a new `wukong application logs` command
- [x] use `yup-oauth2` crate to handle google login and refresh token management
- [x] integrate with https://github.com/mindvalley/wukong-api-proxy/pull/74 API

[PXP-500]: https://mindvalley.atlassian.net/browse/PXP-500?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Review by onimsha — CHANGES_REQUESTED** [review]

File: src/commands/application/logs.rs
Comment (onimsha) [inline src/commands/application/logs.rs:200]: also, we should print out the filter in `TRACE` log level so that we can double confirm what was being sent to Google Logging API when using the `-vvvv` flag.

Comment (jk-gan) [reply src/commands/application/logs.rs:318]: ok, I'll add this

Comment (jk-gan) [reply src/commands/application/logs.rs:320]: Added

**Comment by jk-gan** [comment]
> "07:33:48.450 [info] POST /graphql"

Fixed

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/91#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/91#pullrequestreview-1425749436" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
